_topics/en/swap-in-potentiam.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Swap-in Potentiam (SIP)
+
+## Optional. test Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+shortname: swap-in potentiam
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Contract Protocols
+  - Lightning Network
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Swap-in-Potentiam (SIP)** is a protocol designed to enable the immediate transfer
+  of already-confirmed onchain Bitcoin funds to the Lightning Network, lowering
+  trust requirements by precommitting to an LSP and incorporating a timeout
```

Comment:
"Lowering trust requirements" compared to what?  (I'm guessing compared to zero-conf channels that don't use SIP.)  Perhaps an alternative phrasing could be: "...to the Lightning Network, allowing channels to be opened without either delay or counterparty risk by precommitting..."

Reply:
Right, but I think we need to explain to the reader what baseline trust model we're comparing this to is when we make a claim that this lowers trust requirements.  SIP isn't lower trust than typical channel opens (which wait for multiple confirmations) or onchain spending (which also waits for multiple confirmations); it's only lower trust than an LSP who would otherwise accept zero-conf or low-conf channels.